samples/openthread/coprocessor/prj-rcp_minimal.conf

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable networking
+CONFIG_NETWORKING=y
+
+# Enable OpenThread
+CONFIG_NET_L2_OPENTHREAD=y
+
+# Set OpenThread Radio Co-Processor architecture
+CONFIG_OPENTHREAD_COPROCESSOR=y
+CONFIG_OPENTHREAD_COPROCESSOR_RCP=y
+
+# Use basic 802154 service layer
+CONFIG_NRF_802154_SL_OPENSOURCE=y
+
+# Use built-in mbedtls
+CONFIG_OPENTHREAD_NRF_SECURITY=n
+CONFIG_OPENTHREAD_MBEDTLS=y
+CONFIG_MBEDTLS_HEAP_SIZE=0
```

Comment:
Would it make sense to simply disable mbedTLS heap if it's not needed? (`CONFIG_MBEDTLS_ENABLE_HEAP=n`)

Reply:
Disabling ``MBEDTLS_ENABLE_HEAP`` has no effect as it is selected by ``OPENTHREAD_MBEDTLS``. To fix this I would change _select_ to _imply_ in https://github.com/zephyrproject-rtos/zephyr/blob/1ce264ecdc1facb932efa53f3cbdf8fea45e9727/subsys/net/l2/openthread/Kconfig#L156
What do you think?

Reply:
Thanks for clarifying. I guess it would make sense to switch to imply, as now we have a use case where mbedTLS heap is not needed (it wasn't the case before RCP showed up). But as it's out of the scope of this PR we can stick to the current proposal.